app/src/main/kotlin/com/wire/android/ui/home/newconversation/search/SearchPeopleViewModel.kt

```diff
@@ -0,0 +1,233 @@
+                    label = "test23"
+                ),
+                ExternalContact(
+                    id = 10,
+                    "test1"
+                ), ExternalContact(
+                    id = 11,
+                    "test2",
+                    label = "test235"
+                ), ExternalContact(
+                    id = 12,
+                    "test3",
+                    label = "test234"
+                ),
+                ExternalContact(
+                    id = 13,
+                    "test1"
+                ), ExternalContact(
+                    id = 14,
+                    "test2",
+                    label = "test23"
+                ), ExternalContact(
+                    id = 15,
+                    "This is last contact",
+                    label = "test2"
+                )
+            ),
+            federatedContactSearchResult = listOf(
+                ExternalContact(
+                    id = 1,
+                    "This is first contact",
+                    label = "test23"
+                ), ExternalContact(
+                    id = 2,
+                    "test2",
+                    label = "test23"
+                ), ExternalContact(
+                    id = 3,
+                    "test3",
+                    label = "test23"
+                ),
+                ExternalContact(
+                    id = 4,
+                    "test1",
+                    label = "test23"
+                ), ExternalContact(
+                    id = 5,
+                    "test2",
+                    label = "test23"
+                ), ExternalContact(
+                    id = 6,
+                    "test3",
+                    label = "test23"
+                ),
+                ExternalContact(
+                    id = 7,
+                    "test1",
+                    label = "test23"
+                ), ExternalContact(
+                    id = 8,
+                    "test2",
+                    label = "test23"
+                ), ExternalContact(
+                    id = 9,
+                    "test3",
+                    label = "test23"
+                ),
+                ExternalContact(
+                    id = 10,
+                    "test1"
+                ), ExternalContact(
+                    id = 11,
+                    "test2",
+                    label = "test235"
+                ), ExternalContact(
+                    id = 12,
+                    "test3",
+                    label = "test234"
+                ),
+                ExternalContact(
+                    id = 13,
+                    "test1"
+                ), ExternalContact(
+                    id = 14,
+                    "test2",
+                    label = "test23"
+                ), ExternalContact(
+                    id = 15,
+                    "This is last contact",
+                    label = "test2"
+                )
+            ),
```

Comment:
Tip:
This increases line count dramatically, making the PRs look way scarier to review than it actually is.

Could use a list builder to avoid this.